src/main.js

```diff
@@ -1,4 +1,18 @@
-import Presenter from './presenter/presenter.js';
+import MainPresenter from './presenter/main-presenter.js';
+import HeaderPresenter from './presenter/header-presenter';
+import PointModel from './model/point-model.js';
 
-const presenter = new Presenter();
-presenter.init();
+const pointModel = new PointModel();
+pointModel.init();
+
+const main = document.querySelector('.trip-events');
+
+const mainPresenter = new MainPresenter({
+  boardMainContainer: main,
+  pointModel: pointModel
```

Comment:
Nit: можно сократить 
```suggestion
  pointModel
```